Fix: added missing g color component

AreaScout opened the pull request:
Already commented here b30fe7b4c600eb8ad356f96dcf59150cef118f7a

Commit:
Fix: added missing g color component

dalerank commented:
Thanks a lot

On Sat, 25 Feb 2023, 17:18 Daniel Mehrwald, ***@***.***>
wrote:

> Already commented here b30fe7b
> <https://github.com/dalerank/nanogui-sdl/commit/b30fe7b4c600eb8ad356f96dcf59150cef118f7a>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/dalerank/nanogui-sdl/pull/65
> Commit Summary
>
>    - d79e690
>    <https://github.com/dalerank/nanogui-sdl/pull/65/commits/d79e690c539110f81bdca61c77bce35a60078fe6>
>    Fix: added missing g color component
>
> File Changes
>
> (1 file <https://github.com/dalerank/nanogui-sdl/pull/65/files>)
>
>    - *M* sdlgui/common.cpp
>    <https://github.com/dalerank/nanogui-sdl/pull/65/files#diff-7eeca1d69d028eab33a27e62f7a8aa5ab59b2a8ca4f003a7857ef04479d0ea56>
>    (1)
>
> Patch Links:
>
>    - https://github.com/dalerank/nanogui-sdl/pull/65.patch
>    - https://github.com/dalerank/nanogui-sdl/pull/65.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dalerank/nanogui-sdl/pull/65>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHAEQPEANF3QFSOODWPRTLWZIWFJANCNFSM6AAAAAAVH56QMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

AreaScout commented:
You are welcome

 

Von: dalerank ***@***.***> 
Gesendet: Samstag, 25. Februar 2023 17:21
An: dalerank/nanogui-sdl ***@***.***>
Cc: Daniel Mehrwald ***@***.***>; Author ***@***.***>
Betreff: Re: [dalerank/nanogui-sdl] Fix: added missing g color component (PR #65)

 

Thanks a lot

On Sat, 25 Feb 2023, 17:18 Daniel Mehrwald, ***@***.*** <mailto:***@***.***> >
wrote:

> Already commented here b30fe7b
> <https://github.com/dalerank/nanogui-sdl/commit/b30fe7b4c600eb8ad356f96dcf59150cef118f7a>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/dalerank/nanogui-sdl/pull/65
> Commit Summary
>
> - d79e690
> <https://github.com/dalerank/nanogui-sdl/pull/65/commits/d79e690c539110f81bdca61c77bce35a60078fe6>
> Fix: added missing g color component
>
> File Changes
>
> (1 file <https://github.com/dalerank/nanogui-sdl/pull/65/files>)
>
> - *M* sdlgui/common.cpp
> <https://github.com/dalerank/nanogui-sdl/pull/65/files#diff-7eeca1d69d028eab33a27e62f7a8aa5ab59b2a8ca4f003a7857ef04479d0ea56>
> (1)
>
> Patch Links:
>
> - https://github.com/dalerank/nanogui-sdl/pull/65.patch
> - https://github.com/dalerank/nanogui-sdl/pull/65.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dalerank/nanogui-sdl/pull/65>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHAEQPEANF3QFSOODWPRTLWZIWFJANCNFSM6AAAAAAVH56QMQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.*** <mailto:***@***.***> >
>



—
Reply to this email directly, view it on GitHub <https://github.com/dalerank/nanogui-sdl/pull/65#issuecomment-1445152053> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAM2RGB2Y3NUPQQ5RMZOQ5LWZIWP3ANCNFSM6AAAAAAVH56QMQ> .
You are receiving this because you authored the thread.  <https://github.com/notifications/beacon/AAM2RGDDLDMHASMGRDTJMRTWZIWP3A5CNFSM6AAAAAAVH56QMSWGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTSWENCTK.gif> Message ID: ***@***.*** ***@***.***> >

AreaScout commented:
You are welcome
